tools/serverpod_cli/bin/downloads/resource_manager.dart

```diff
@@ -13,41 +14,40 @@ class ResourceManager {
   Directory get homeDirectory {
     var envVars = Platform.environment;
 
-    if (Platform.isMacOS) {
-      return Directory(envVars['HOME']!);
-    } else if (Platform.isLinux) {
-      return Directory(envVars['HOME']!);
-    } else if (Platform.isWindows) {
+    if (Platform.isWindows) {
       return Directory(envVars['UserProfile']!);
+    } else if (!Platform.isWindows) {
+      return Directory(envVars['HOME']!);
+    } else {
```

Comment:
This case can never be hit. Better to use an if / else or define all supported platforms Mac/Linux/Windows

Reply:
Not sure why this code was changed actually?

Reply:
remove else.